docs/developer/tutorials/snmp/profiles.md

```diff
@@ -79,6 +79,91 @@ metrics:
 !!! tip
     `sysobjectid` can also be a wildcard pattern to match a sub-tree of devices, eg `1.3.6.1.131.12.4.*`.
 
+### Generate a profile file from a collection of MIBs
+
+You can use `ddev` to create a profile from a list of mibs.
+
+```console
+$  ddev meta snmp generate-profile-from-mibs --help
+```
+
+This script requires a list of ASN1 MIB files as input argument, and copies to the clipboard a list of metrics that can be used to create a profile.
+
+#### Options
+
+`-f, --filter-path-oids` is an option to provide the path to a YAML file to be used as a filter to exclude some nodes within a MIB.
```

Comment:
> is an option to provide the path to a YAML file to be used as a filter to exclude some nodes within a MIB.

Not sure to understand "filter to exclude some nodes", when I tested the tool, it seems we are including the symbols that are listed.

I tested with this filter:

```
IF-MIB:
- ifNumber
```

The output of:

```
$ ddev meta snmp generate-profile-from-mibs  IF-MIB --filter-path-oids filter.yaml
```

is 

```
metrics:
- MIB: IF-MIB
  symbol:
    name: ifNumber
    OID: 1.3.6.1.2.1.2.1.0
    description: The number of network interfaces (regardless of their current state)
      present on this system.
```

Reply:
Indeed, items listed in filters are included. I will rephrase the documentation to avoid misunderstandings